Trimble.Modus.Components/Handlers/EditorHandler.cs

```diff
@@ -22,7 +22,7 @@ public static void MapTMEditor(IEditorHandler editorHandler, IEditor editor)
 #if ANDROID
                 editorHandler.PlatformView.SetPadding(0, 0, 0, 0);
                 editorHandler.PlatformView.Background = null;
-                editorHandler.PlatformView.BackgroundTintList = Android.Content.Res.ColorStateList.ValueOf(Colors.Transparent.ToAndroid());

```

Comment:
Colors.Transparent is missing in Android?

Reply:
Colors.Transparent.ToAndroid() this not support for .net 9 apps, hence i have changed to Android.Graphics.Color.Transparent